content/en/cloud_cost_management/cost_allocation/bigquery.md

```diff
@@ -0,0 +1,112 @@
+
+## Resources
+CCM allocates costs for BigQuery resources for both types of Analysis costs (on-demand and reservation-based). CCM displays costs for resources including query-level analysis, storage, and data transfer on the [**BigQuery** dashboard][1].
+
+## Prerequisites
+
+The following table presents the list of collected features and the minimal requirements:
+
+| Feature | Requirements |
+|---|---|
+| Retrieve tags from labels of a query | Supported without monitoring or reservations |
+| Query-Level Cost Attribution | BigQuery monitoring enabled |
+| Reservation Cost Allocation | BigQuery reservations configured |
+
+1. Configure the Google Cloud Cost Management integration on the [Cloud Cost Setup page][2].
+2. Enable BigQuery monitoring in your Google Cloud project.
+3. For reservation cost allocation, configure BigQuery reservations in your project.
+
+## Allocating costs
+
+Cost allocation divides BigQuery costs from GCP into individual queries and workloads associated with them. These divided costs are enriched with tags from queries, projects, and reservations so you can break down costs by any associated dimensions. 
+
+For reservation-based BigQuery costs, CCM allocates costs proportionally based on slot usage. Each query's cost is determined by its share of the total slot usage within the project's reservations. For example, if a query uses 25% of the total consumed slots in a project's reservation during a given period, it will be allocated 25% of that project's total reservation cost for that period. The cost per-query is calculated using the following formula:
+
+```
+cost_per_query = (query_slot_usage / total_slot_usage) * total_project_reservation_cost
+```
+
+Where:
+- `query_slot_usage`: The number of slot-seconds consumed by an individual query
+- `total_slot_usage`: The total slot-seconds used across all queries in the project's reservations
+- `total_project_reservation_cost`: The total cost of the reservations in a given project for the time period
+
+Any difference between the total billed reservation cost and the sum of allocated query costs is categorized as a project's `cluster_idle` cost, representing unused reservation capacity. 
+
+### Understanding idle costs
+
+Idle costs represent the portion of reservation capacity that was paid for but not utilized by queries. These costs arise when the reserved slot capacity exceeds actual usage during a billing period.
+
+**Idle slot sharing considerations**: If your organization has enabled idle slot sharing between reservations, the idle cost calculation may appear different than expected. When queries from one project use idle slots from another project's reservation, those slot costs are attributed as "free" rather than to the consuming project. This means:
```

Comment:
Is there a GCP doc we can link here with more info on how this is set up? Could be a good source to add